src/SonarLint.VSSpecificAssemblies.props

```diff
@@ -143,4 +143,16 @@
     <StrongNamerKeyFile>$(AssemblyOriginatorKeyFile)</StrongNamerKeyFile>
   </PropertyGroup>
 
+
+  <!-- ******************************************************************************* -->
+  <!-- VCProject assembly -->
+  <!-- ******************************************************************************* -->
+  <ItemGroup Label="VCProjectEngine" Condition="$(RequiresVCProjectEngine)=='true'">
```

Comment:
Out of curiosity, why not always add the assembly? Why have the "RequiresVCProjectEngine" condition?

Reply:
So we know which projects require the assembly and which ones don't. It's a bad practice to reference assemblies that aren't required, and this is a common settings file that is imported in multiple projects.